<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>501</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="title">
     <string>Interfaces mit Internet</string>
    </property>
    <widget class="QLineEdit" name="INTERFACE_NET">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="INTERFACE_NET_mac">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mac</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>501</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="title">
     <string>ap-hotspot settings</string>
    </property>
    <widget class="QLineEdit" name="INTERFACE">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="INTERFACE_mac">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="driver">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ssid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="hw_mode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="channel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="macaddr_acl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="auth_algs">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="ignore_broadcast_ssid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="wpa">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="wpa_passphrase">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="wpa_key_mgmt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="wpa_pairwise">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>dnsmasq.d</string>
    </property>
    <widget class="QLineEdit" name="rsn_pairwise">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="bind_interfaces">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="dhcp_range">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
